test/correctness/gpu_mixed_shared_mem_types.cpp

```diff
@@ -46,7 +47,9 @@ int main(int argc, char **argv) {
         int off = 0;
         if ((types[i].is_int() || types[i].is_uint())) {
             // Metal does not support 64-bit integers.
```

Comment:
Can we use the target supports type functionality here?

Reply:
Done!